tests/test_TUDPTransport.py

```diff
@@ -14,9 +14,10 @@
 
 from __future__ import absolute_import
 
-import mock
 import unittest
 
+import mock
```

Comment:
I don't get it. Are there actually some rules it uses? Like built-in modules first? That might explain the break on py3